### ISSUE TITLE
"callHandler" method is not injected into InAppBrowser

### ISSUE THREAD

**Issue by 7akub** [pichillilorenzo/flutter_inappwebview#1973]
- [x] I have read the [Getting Started](https://inappwebview.dev/docs/intro/) section
- [x] I have already searched for the same problem

## Environment

| Technology            | Version       |
|-----------------------| ------------- |
| Flutter version       |     3.13.2          |
| Plugin version        |     6.0.0          |
| Android version       |      11, MIUI 12.5.15/12.1.1         |
| iOS version           |      n/a         |
| macOS version         |      n/a         |
| Xcode version         |        n/a      |
| Google Chrome version |     n/a         |

Device information: Xiaomi Redmi Note 8 Pro / Xiaomi Mi 9T

## Description
Provided code worked fine before I moved from version `5.8.0` to version `6.0.0`. After upgrade `callHandler` is not available from the website opened in `InAppBrowser`. I've been looking for a solution, but it seems that no one else have been affected by this.

Stoped working only on Android devices, I tested it on iOS 17.1.2 and it works just fine.

**Expected behavior:** 
`callHandler` method should be injected into `InAppBrowser`

**Current behavior:** 
`callHandler` is undefined when creating `InAppBrowser`

## Steps to reproduce
```dart
final options = InAppBrowserClassSettings(
  browserSettings: InAppBrowserSettings(
    hideTitleBar: true,
    allowGoBackWithBackButton: false,
    shouldCloseOnBackButtonPressed: true,
    hideToolbarTop: !displayNavigation,
    hideUrlBar: true,
  ),
  webViewSettings: InAppWebViewSettings(
    useShouldOverrideUrlLoading: true,
    javaScriptEnabled: true,
    isInspectable: true,
    clearCache: true,
    clearSessionCache: true,
  ),
);

final InAppBrowser browser = CustomBrowser(
  onCloseCallback: () {
    // Do something
  },
);
  
browser.openUrlRequest(
urlRequest: URLRequest(url: WebUri(url)),
settings: options,
);
```

```dart
class CustomBrowser extends InAppBrowser {
  CustomBrowser({
    this.onCloseCallback,
    int? windowId,
    UnmodifiableListView<UserScript>? initialUserScripts,
  }) : super(
          windowId: windowId,
          initialUserScripts: initialUserScripts,
        );

  final Function()? onCloseCallback;

  @override
  Future onBrowserCreated() async {
    developer.log("Browser Created!");

    if (webViewController != null) {
      developer.log('Adding JS handler');

      webViewController!.addJavaScriptHandler(
        handlerName: JavaScriptHandlerType.closeInAppBrowser,
        callback: (_) {
          if (onCloseCallback != null) {
            onCloseCallback!();
          }

          close();
        },
      );
    }
  }
}
```

## Images   
![image](https://github.com/pichillilorenzo/flutter_inappwebview/assets/106750898/b7e93f9f-39fe-438f-bd92-212906518026)

![image](https://github.com/pichillilorenzo/flutter_inappwebview/assets/106750898/a41d90b1-0e3f-44bb-bb2a-f671dd9a333a)

**Comment by fuzzybinary** [comment]
I'm also being affected by this. 

It appears to be an issue in Android 33+ where `WebViewFeature.DOCUMENT_START_SCRIPT` is supported. From what I can tell, there's no point at which `JAVASCRIPT_BRIDGE_JS_SOURCE` is being injected when this feature is enabled.

When testing on Android 31, `prepareAndAddUserScripts` is called as part of `InAppWebview.prepare`. but it is not called on Android 33+, and `prepareAndAddUserScripts` appears to be the only location where JAVASCRIPT_BRIDGE_JS_SOURCE could be injected.

**Comment by pichillilorenzo** [comment]
Released new version `6.2.0-beta.2` with the fix.
